### ISSUE TITLE
IKEA Nymåne pendant lamp

### ISSUE THREAD

**Issue by cklit** [dresden-elektronik/deconz-rest-plugin#6230]
## Device

- Product name: Ikea Nymåne (pendant)
- Manufacturer: IKEA of Sweden
- Model identifier: NYMANE PENDANT
- Device type : 
  - Light

## Screenshots

<img width="415" alt="image" src="https://user-images.githubusercontent.com/23294994/181630521-468f318a-4d05-49b9-a376-9752ee76a573.png">

<img width="245" alt="image" src="https://user-images.githubusercontent.com/23294994/181630616-13b7e8b7-ee68-4d49-b24e-d8c09787b326.png">

### Basic
<img width="409" alt="image" src="https://user-images.githubusercontent.com/23294994/181630474-e1b78812-b9f0-459e-ad1a-5fabea35fb54.png">

### Identify
<img width="400" alt="image" src="https://user-images.githubusercontent.com/23294994/181630443-e4adf193-383b-483e-afa2-a68bf1d18279.png">

### Groups
<img width="382" alt="image" src="https://user-images.githubusercontent.com/23294994/181630748-92fbb8db-12bc-4e59-a5c5-88f79b9e66cd.png">

### Scenes
<img width="387" alt="image" src="https://user-images.githubusercontent.com/23294994/181630848-19886072-e8c3-4a3d-8072-434aaf0f20c6.png">

### On/Off
<img width="450" alt="image" src="https://user-images.githubusercontent.com/23294994/181630986-1af67b15-d9ed-439c-b940-c1e7dec2e00a.png">

### Level Control
<img width="445" alt="image" src="https://user-images.githubusercontent.com/23294994/181631025-635dceec-04d9-4e78-83f6-fddd27aac8ed.png">

### Color Control
<img width="448" alt="image" src="https://user-images.githubusercontent.com/23294994/181631094-6a5dfefd-e5c0-4c18-bf90-ad1007250d6d.png">

### IKEA specific
<img width="462" alt="image" src="https://user-images.githubusercontent.com/23294994/181631354-4b5b64bd-3e48-491a-9221-3a061129502b.png">

### Green power
<img width="456" alt="image" src="https://user-images.githubusercontent.com/23294994/181631394-4f7b263b-466b-4d96-b9d9-ff3f93f2fbe5.png">

**Comment by Mimiix** [comment]
Hi,

What is the issue with it? Isn't it already working?

**Comment by cklit** [comment]
Not for me :/ Unable to control it from Home Assistant, and does not show up in Phoscon either. I have 2 lamps, its the same with both, and I've tried to reset multiple times.

**Comment by cacedo** [comment]
It's working for me. I reset it by switching on/off 6 times, then add device in Home Assistant

**Comment by cklit** [comment]
It might have been a local issue. I tried once again to reset the bulb (6x on/off) and search for the lights. Usually new lights show up in the bottom of the list in Phoscon, but these did not for whatever reason. But I could see in the deconz vnc panel that it found the lamps and named them "Color temperature light <x>". Once I refreshed the page in Phoscon I could see "Color temperature light" mixed together with all my other lights.

They behaved strange for about 2 mins (verrrrrry slow response, but only those 2 lights), which leads me to think that it may be something locally with my network, but now they both work perfectly. 

Closing.
